liquid_node/backup.py

```diff
@@ -228,9 +240,9 @@ def backup_pg(dest_file, username, dbname, alloc):
     tmp_file = Path(str(dest_file) + '.tmp')
     log.info(f"Dumping postgres from alloc {alloc} user {username} db {dbname} to {tmp_file}")
     cmd = (
-        f"set -exo pipefail; ./liquid dockerexec {alloc} "
-        f"pg_dump -U {username} {dbname} -Ox "
-        f"| gzip -1 > {tmp_file}"
+        f'set -exo pipefail; ./liquid dockerexec {alloc} '
+        f'pg_dump {databases_to_ignore_str()} -U {username} {dbname} -Ox '
```

Comment:
this applies this complication to all postgres databases.

we don't want to do that (even if we know for sure there'll be no table with the same name in the future) because it makes the whole process more convoluted and confusing to debug

plz only apply the ignore option to the actual database we want to work with (snoop)

Reply:
maybe some `ignore_tables = []` optional arg to the `backup_pg(...)` function